crypto.go

```diff
@@ -53,3 +92,121 @@ func GenerateSigningKeys() (SigningKeys, error) {
 	}
 	return signingKeys, err
 }
+
+// EncryptData accepts a record object of plain meta and plain data, returning
+// a record with data keys encrypted with dk and ak according to Tozstore spec.
```

Comment:
Is the spec something that could be linked here?

Reply:
Not sure if there is an internal spec outlined anywhere